backend/src/main/java/ch/zuehlke/fullstack/hackathon/integration/EmployeeClient.java

```diff
@@ -0,0 +1,41 @@
+package ch.zuehlke.fullstack.hackathon.integration;
+
+import ch.zuehlke.fullstack.hackathon.domain.UserInfo;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+@Service
+public class EmployeeClient {
```

Comment:
Should we call it EmployeeService? We could also think about splitting client and service then, e.g.

- client is responsible for calls to insight API (rename to InsightClient)
- service is responsible for gathering info from all needed endpoints

Reply:
Renamed it to InsightClient and did not yet add an EmployeeService (since it has no logic yet).